server/channelserver/handlers_cast_binary.go

```diff
@@ -391,7 +397,7 @@ func handleMsgSysCastBinary(s *Session, p mhfpacket.MHFPacket) {
 			bf.SetLE()
 			chatMessage := &binpacket.MsgBinChat{}
 			chatMessage.Parse(bf)
-			if strings.HasPrefix(chatMessage.Message, "!") {
+			if strings.HasPrefix(chatMessage.Message, s.server.erupeConfig.CommandPrefix) {
```

Comment:
The first line of `parseChatCommand` splits on index 1 so need to check for the prefix length.
```go
args := strings.Split(command[1:], " ")
```

Reply:
Good catch.